packages/p2p/src/components/advertiser-page/block-user/block-user-modal.jsx

```diff
@@ -0,0 +1,72 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { observer } from 'mobx-react-lite';
+import { useStores } from 'Stores';
+import { Localize } from 'Components/i18next';
+
+const BlockUserModal = ({ is_advertiser_blocked, onCancel, onSubmit }) => {
+    const { advertiser_page_store } = useStores();
+
+    return (
+        <Modal
+            is_open={advertiser_page_store.is_block_user_modal_open}
```

Comment:
Since this is a reusable component, you should add a prop `is_modal_open` to make it flexible to use across advertiser's page and my profile